Find a functionally equivalent code:android.os.Process.myPpid:COMMENT
Method Modifier: public      static      final       hidden      
Comment:/**
 * Returns the identifier of this process' parent.
 * @hide
 */

@@ -1,3 +1,3 @@
 {
-    return Libcore.os.getppid();
+    return Os.getppid();
 }

